src/position/docs/readme.md

```diff
@@ -1,114 +1,182 @@
 The `$uibPosition` service provides a set of DOM utilities used internally to absolute-position an element in relation to another element (tooltips, popovers, typeaheads etc...).
 
-### getRawNode(element)
-  Takes a jQuery/jqLite element and converts it to a raw DOM element.
+#### getRawNode(element)
+
+Takes a jQuery/jqLite element and converts it to a raw DOM element.
 
 ##### parameters
 
-* `element` _(Type 'object')_ - The element to convert.
+* `element`
+  _(Type: `object`)_ -
+  The element to convert.
 
 ##### returns
 
-  _(Type 'element')_ - A raw DOM element.
+* _(Type: `element`)_ -
+  A raw DOM element.
+
+#### parseSyle(element)
 
-### parseSyle(element)
-  Parses a numeric style value to a number.  Strips units and will return 0 for invalid (NaN) numbers.
+Parses a numeric style value to a number.  Strips units and will return 0 for invalid (NaN) numbers.
 
 ##### parameters
 
-* `value` _(Type 'string')_ - The style value to parse.
+* `value`
+  _(Type: `string`)_ -
+  The style value to parse.
```

Comment:
This function is used to parse any style value that may have a unit to a number, e.g. width: 5px to 5.